nix/overlays.nix

```diff
@@ -28,10 +28,12 @@ in {
     (final: prev: let
       date = mkDate (self.lastModifiedDate or "19700101");
     in {
-      hyprland = prev.callPackage ./default.nix {
+      hyprland = final.callPackage ./default.nix {
         stdenv = prev.gcc13Stdenv;
         version = "${props.version}+date=${date}_${self.shortRev or "dirty"}";
+        wlroots = prev.wlroots-hyprland;
         commit = self.rev or "";
+        inherit (prev) udis86 hyprland-protocols;
```

Comment:
this is also using `prev`. Could rename the output to `hyprland-udis86`, and then `udis86 = final.hyprland-udis86`

Reply:
Same as above, I think you're somehow seeing an outdated version.